vpr/src/base/vpr_context.h

```diff
@@ -133,6 +133,10 @@ struct DeviceContext : public Context {
     std::vector<t_physical_tile_type> physical_tile_types;
     std::vector<t_logical_block_type> logical_block_types;
 
+    /* Boolean that indicates whether the architecture implements an N:M
+     * physical tiles to logical blocks mapping */
+    bool has_multiple_equivalent_tiles;
```

Comment:
Does this ever get used?

Reply:
This is actually a workaround I have used to disable sorting of the blocks before the initial placement. It is currently used, but I still need to figure out a way to not be dependent on this.

Reply:
This is actually a workaround I have used to disable sorting of the blocks before the initial placement. It is currently used, but I still need to figure out a way to not be dependent on this.